doc/03_research/03_planning/Implementation.md

```diff
@@ -0,0 +1,135 @@
+# Planning Implementation
+
+**Summary:** 
+The document gives a first impression of how the planning could/should work
+and how the topics are edited
+---
+
+## Author
+
+Simon Erlbacher, Niklas Vogel
+
+## Date
+
+29.11.2022
+
+## Prerequisite
+
+
+---
+<!-- TOC -->
+* [Planning Implementation Details](#coding-style-guidelines)
+  * [Author](#author)
+  * [Date](#date)
+  * [Prerequisite](#prerequisite)
+  * [Visualization](#visualization)
+  * [Preplanning](#preplanning)
+  * [Local Path Planning](#local-path-planning)
+  * [Decision Making](#decision-making)
+  * [Next steps](#next-steps)
+* [Sources](#sources)
+<!-- TOC -->
+
+---
+## **Visualization** 
```

Comment:
Warum sind Überschriften in diesem Dokument zusätzlich fett oder kursiv geschrieben?